src/main/java/gregtech/api/metatileentity/MetaTileEntity.java

```diff
@@ -1236,4 +1236,7 @@ public boolean keepsInventory() {
     public boolean getWitherProof() {
         return false;
     }
+
+    public void preInit(Object... data) {
+    }
```

Comment:
I see your point, but if the method is only called by your `MetaTileEntityClipboard`, I don't think it's worth making changes at the abstraction level. Why not make this a unique method of `MetaTileEntityClipboard`? Call it when you create a MTE in ClipboardBehavior's code

Reply:
After thinking about how to go about implementing this, I actually tried to do this, and I created a new clipboard for me to modify before assigning it to the MetaTileEntityHolder. However, the problem is that in the process, several new null clipboards were somehow actually placed in-world to nearby positions for extremely strange reasons. In fact, one of them was constantly playing the "player using item" sound. The cause of such a glitch is probably out of my scope for now, and as such, I think that I'm going to have to leave it like this.